pkg/controller.v2/tfcontroller/tfcontroller.go

```diff
@@ -75,7 +67,7 @@ var (
 // TFJobController is the type for TFJob Controller, which manages
 // the lifecycle of TFJobs.
 type TFJobController struct {
-	jobcontroller.JobController
+	*jobcontroller.JobController
```

Comment:
I am not sure why using pointer here, could you please explain more about it?

Reply:
No modification. Hence not necessary. I will update it.

Reply:
Thanks